### ISSUE TITLE
WCAG 2.2 new zoom level cutting out content on top,sides and bottom of streaming serivcre videos

### ISSUE THREAD

**Issue by kyledalton16-hub** [w3c/wcag#4867]
WCAG 2.2 standards, specifically those aimed at improving accessibility for users with low vision (such as requirements for 400% zoom and larger interactive targets can cause layout issues where content appears too zoomed in, crowded, or improperly scaled on streaming service interfaces. While intended to make services more accessible, the implementation has led to frustrations where interface elements are too large or, conversely, where content is not properly responsive, leading to unintended zoom or, in some cases, content being cut off., this frusrtriang no one sked for this reidcoucs zoom level, i have not be able to use my paid subscriptions beauces of this

**Comment by patrickhlauke** [comment]
> can cause layout issues

when the authors/developers don't apply them correctly. if content, as a result of it, is cropped/unusable, that fails the requirements already, so it's those streaming services that are doing it wrong/failing.

**Comment by kyledalton16-hub** [comment]
i don't think you quite understood what i said if the media isnt made with
these new ratios in mind, which is a lot, zooming out is not the only thing
that cut content off but 2-000400% scale zoom level your reuerre is
extreme, if its was minimum was set to 110% to 120 it be manual, and of
course streaming services are that have wen version will apply same
coding to all their version which will result in errors, no i know  in
disability community like this, or wanted this, what a lot people wanted is
accurate cc, ability to increase audio or change pitch, and accurate type
to text software vice versa, so according to you this  while violets this
new wcag2.2, the fault lies with streaming services, did you even into take
into account the technical limitations of increasing zoom for certain
display screen, so since your saying its on their end can you
provide example or instrcutrtion on how the zoom meets minimum target level
without ridiculously cropping videos

Kind Regards
KyleDalton

**Comment by mraccess77** [comment]
WCAG 2.2 SC 1.4.10 doesn't require 400% zoom.  It requires content not require scrolling in 2 dimensions at 320 CSS pixels.  It doesn't actually require anything enlarge.

**Comment by TestPartners** [comment]
> WCAG 2.2 SC 1.4.10 doesn't require 400% zoom. It requires content not require scrolling in 2 dimensions at 320 CSS pixels. It doesn't actually require anything enlarge.

The normative text does not refer to "scrolling in 2 dimensions **at** 320 CSS pixels". It refers to "Vertical scrolling content at a width **equivalent to** 320 CSS pixels". Furthermore, all the non-normative notes and the Understanding page refer to zooming and text enlarging. It is very clear that the authors of the SC did not intend it to be tested at 100% zoom - they could have said "at or equivalent to", but they didn't.

Issues such as floating elements obscuring other content are far less apparent at 100% zoom and a 320px wide window than in a 1208px window at 400% zoom. I would therefore regard the former as a bad way to do testing even if it was permitted (which I assert it is not).

**Comment by kyledalton16-hub** [comment]
ir require pixel size to be increased and new minum target is 400%zoom if
that not correcert then its wording is misleading is causing the
problems, (Level
AA)

Content can be presented without loss of information or functionality, and
without requiring scrolling in two dimensions for:

   - Vertical scrolling content at a width equivalent to 320 CSS pixels
   <https://www.w3.org/TR/WCAG22/#dfn-css-pixels>;
   - Horizontal scrolling content at a height equivalent to 256 CSS pixels
   <https://www.w3.org/TR/WCAG22/#dfn-css-pixels>.

Except for parts of the content which require two-dimensional layout for
usage or meaning.
Note 1

320 CSS pixels is equivalent to a starting viewport
<https://www.w3.org/TR/WCAG22/#dfn-viewport> width of 1280 CSS pixels wide
at 400% zoom. For web content which is designed to scroll horizontally
(e.g., with vertical text), 256 CSS pixels is equivalent to a starting
viewport height of 1024 CSS pixels at 400% zoom.
Note 2

Examples of content which requires two-dimensional layout are images
required for understanding (such as maps and diagrams), video, games,
presentations, data tables (not individual cells), and interfaces where it
is necessary to keep toolbars in view while manipulating content. It is
acceptable to provide two-dimensional scrolling for such parts of the
content.
Success Criterion 1.4.11 Non-text Contrast
<https://www.w3.org/TR/WCAG22/#non-text-contrast>
Understanding Non-text Contrast
<https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html>|
How to Meet Non-text Contrast
<https://www.w3.org/WAI/WCAG22/quickref/#non-text-contrast>

(Level AA)

The visual presentation <https://www.w3.org/TR/WCAG22/#dfn-presentation> of
the following have a contrast ratio
<https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio> of at least 3:1 against
adjacent color(s):
User Interface ComponentsVisual information required to identify user
interface components
<https://www.w3.org/TR/WCAG22/#dfn-user-interface-components> and states
<https://www.w3.org/TR/WCAG22/#dfn-states>, except for inactive components
or where the appearance of the component is determined by the user agent
<https://www.w3.org/TR/WCAG22/#dfn-user-agents> and not modified by the
author;Graphical ObjectsParts of graphics required to understand the
content, except when a particular presentation of graphics is essential
<https://www.w3.org/TR/WCAG22/#dfn-essential> to the information being
conveyed.as righ there size is being told to be increased


Understanding Visual Presentation
<https://www.w3.org/WAI/WCAG22/Understanding/visual-presentation.html>|
How to Meet Visual Presentation
<https://www.w3.org/WAI/WCAG22/quickref/#visual-presentation>

(Level AAA)

For the visual presentation of blocks of text
<https://www.w3.org/TR/WCAG22/#dfn-blocks-of-text>, a mechanism
<https://www.w3.org/TR/WCAG22/#dfn-mechanism> is available to achieve the
following:

   - Foreground and background colors can be selected by the user.
   - Width is no more than 80 characters or glyphs (40 if CJK).
   - Text is not justified (aligned to both the left and the right margins).
   - Line spacing (leading) is at least space-and-a-half within paragraphs,
   and paragraph spacing is at least 1.5 times larger than the line spacing.
   - Text can be resized without assistive technology
   <https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies> up to 200
   percent in a way that does not require the user to scroll horizontally to
   read a line of text on a full-screen window
   <https://www.w3.org/TR/WCAG22/#dfn-on-a-full-screen-window>.
   - as you can see and many compines will simply apply that feature or new
   zoom across board as it is cheapset option for them

Note 1

Content is not required to use these values. The requirement is that a
mechanism is available for users to change these presentation aspects. The
mechanism can be provided by the browser or other user agent. Content is
not required to provide the mechanism.
Note 2

Writing systems for some languages use different presentation aspects to
improve readability and legibility. If a presentation aspect in this
success criterion is not used in a writing system, content in that writing
system does not need to use that presentation setting and can conform
without it. Authors are encouraged to follow guidance for improving
readability and legibility of text in their writing system.

**Comment by patrickhlauke** [comment]
>  It is very clear that the authors of the SC did not intend it to be tested at 100% zoom

you can test at 100% if you resize your browser window, or are on a device with a limited size screen, that in the end shakes out to a width of 320 CSS px. not quite sure what you're arguing around here...

in any case, going back to the original question: this isn't going to change in WCAG 2.x.

**Comment by TestPartners** [comment]
> you can test at 100% if you resize your browser window, or are on a device with a limited size screen, that in the end shakes out to a width of 320 CSS px. not quite sure what you're arguing around here...

As I said in my previous post, issues such as floating elements obscuring other content are far less apparent at 100% zoom and a 320px wide window. If you test at that zoom level, you're going to miss issues that affect people who use higher zoom levels. And they are precisely the people who the SC is aimed at. It's not aimed at people with very narrow browser windows or screens.

100% zoom and a 320px wide window is not the same as 400% zoom and a 1280px window unless you reduce the height of the 320px wide window to a quarter of its normal height (and maybe not even then).

It's so trivially easy to do this test properly at 400% zoom, that I truly don't understand why anyone is arguing against doing so.

**Comment by kyledalton16-hub** [comment]
but this new zoom minum ruins layout for almost all forms of media as they
are not designed for all this new target going to do is cut content out or
force people to newer devivces

**Comment by detlevhfischer** [comment]
@TestPartners wrote:

> If you test at that zoom level, you're going to miss issues that affect people who use higher zoom levels. 

I agree that is is useful to test with 400% zoom starting from 1280 viewport width for the reasons you give, and record any issues that arise. Normatively though, the requirement does not say anything about fixed content or about how much space should remain for content to still pass after magnification. It also does not define the viewport height to test at which makes all the difference regarding cover-up issues. You can call it a loophole but that is what is in the normative text. So I think @patrickhlauke  is right to treat 100% at 320px as equivalent in terms of the normative ask of 1.4.10. There may be other SCs such as 2.4.11 Focus Not Obscured that may catch situations where fixed content hides other content.

**Comment by TestPartners** [comment]
> So I think [@patrickhlauke](https://github.com/patrickhlauke) is right to treat 100% at 320px as equivalent in terms of the normative ask of 1.4.10. 

But that's not what the normative text says. It refers to "Vertical scrolling content at a width **equivalent to** 320 CSS pixels". It does not say "**at or equivalent to** 320 CSS pixels".

You are right that the requirement does not say anything about how much space should remain for content to still pass after magnification. But we shouldn't just ignore it - we should make an intelligent subjective judgement, as we do with many other success criteria. And to do that, we should consider how people will view the page - many people use magnification, but I suspect that almost no one uses a 320px wide viewport at 100% zoom.

**Comment by kyledalton16-hub** [comment]
i am saying new target of 400% is resulting layout breaks

**Comment by patrickhlauke** [comment]
> But that's not what the normative text says. It refers to "Vertical scrolling content at a width equivalent to 320 CSS pixels". It does not say "at or equivalent to 320 CSS pixels".

I believe you are trying to make a distinction where it doesn't exist. when you zoom, you DO change the CSS pixel width of the window, and it IS 320 CSS pixels if you started at 1280 and zoomed 400%. the "equivalent" was used to also cover things like PDFs where the unit of measure isn't necessarily CSS pixels (and though nominally not covered by WCAG, environments such as native iOS which uses "points", or Android native development which uses "density-independent pixels" as the measure - which are conceptually equivalent to CSS pixels in that they don't mean actual physical pixels, and are independent of the actual pixel density used)
